cmd/garm/main.go

```diff
@@ -82,6 +84,19 @@ func main() {
 		log.Fatalf("fetching log writer: %+v", err)
 	}
 
+	if cfg.Default.LogFile != "" {
```

Comment:
You don't need to add this if. We need to install this signal handler regardless, otherwise the daemon will exit on sighup.

Reply:
But, if we don't have this `if`, the call:
```go
logWriter.(*lumberjack.Logger).Rotate()
```
from below will fail because `logWriter` will be simply `os.Stdout`, isn't it?